### ISSUE TITLE
zsh: can't set tty pgrp: Input/output error

### ISSUE THREAD

**Issue by lencioni** [sorin-ionescu/prezto#1033]
I recently updated from f2a826e963f06a204dc0e09c05fc3e5419799f52 to 7227c4f0bef5f8ae787c65150d7a7403394fff48 which caused zsh to freeze with the message "zsh: can't set tty pgrp: Input/output error" when starting up. I'm not entirely sure what caused this, but it seems related to one of the submodules. Any ideas?

Running zsh 5.0.5 (x86_64-apple-darwin14.0)

**Comment by sorin-ionescu** [comment]
No idea. I have never seen that error before.

**Comment by lencioni** [comment]
Oh, I should also mention that I saw this while opening new panes and windows in tmux. Opening a new terminal just caused it to close out automatically.

**Comment by sorin-ionescu** [comment]
It might tmux-related, not Zsh, certainly not Prezto.

**Comment by lencioni** [comment]
I am running into this on a new laptop that I am setting up and have pinpointed it to de9ed0610 which updates external syntax-highlighting. I have verified that updating that submodule to its latest master (https://github.com/zsh-users/zsh-syntax-highlighting/commit/73ee7c1f6c4) does not resolve my problem. Using git bisect I was able to determine that my problem was introduced by https://github.com/zsh-users/zsh-syntax-highlighting/commit/59fbdda64 but I don't actually see why. Any thoughts?

If nothing is obvious to you, I can move discussion over to that repo.
